### DISCUSSION TITLE
Multiple file limit while pick file from gallary.

### DISCUSSION THREAD

**Discussion by DharmikGorsiya** [yasirkula/UnityNativeGallery#253] [Q&A]
Hello @yasirkula and Team,
Nice plugin to use.

I just want to know can I add limit for file pic while I am using multiple file from gallary.

**Comment by yasirkula** [comment]
Hi! Please see: https://github.com/yasirkula/UnityNativeGallery/issues/181

**Reply by DharmikGorsiya** [reply]
Hi @yasirkula,
Thanks for quick reply there is any way to do it in Android as of now, or any probability to come in future.

**Reply by yasirkula** [reply]
Unfortunately not because Android's native media picker doesn't have an API for this. Some device vendors might've added their own APIs for this but there isn't a global solution, unfortunately. We must create our own native media picker UI to achieve this but I'm not planning to do that at the moment.
